### PR TITLE
feat: add new package @ui5/middleware-code-coverage

### PR DESCRIPTION
**Pull request by flovogt**
Adds new UI5 middleware to provide client side code coverage determination.

Wait for submitting this PR until UI5 Version 1.113 is released, so the middleware can be actually used.

### CODE REVIEW

**Comment by marianfoo** [comment]
Hey Florian,
Thank you for your PR.
I converted the PR to a draft to show this should not be merged yet.
I just saw in the readme of your package minimal version is 1.114?

**Comment by flovogt** [comment]
> Hey Florian, Thank you for your PR. I converted the PR to a draft to show this should not be merged yet. I just saw in the readme of your package minimal version is 1.114?

Hi Marian,
thanks a lot for your reply and making this PR a draft. Good idea!
I have created a PR to update the readme. UI5 1.113.0 should be fine. https://github.com/SAP/ui5-tooling-extensions/pull/57

File: sources.json [hunk 1]
```diff
@@ -581,5 +581,19 @@
...
     "tags": ["development", "tooling", "testing", "browser"],
     "license": "mit"
+  },
+    {
+    "owner": "SAP",
+    "repo": "ui5-tooling-extensions",
+    "subpath": "packages",
+    "subpackages": [
+      {
+        "name": "@ui5/middleware-code-coverage",
```
Comment (marianfoo) [inline sources.json]: this need to be the folder name "middleware-code-coverage" no the package name
i get the package name from the package.json

File: sources.json [hunk 2]
```diff
@@ -581,5 +581,19 @@
...
+        "addedToBoUI5": "2023-03-22T14:12:29.389Z",
```
Comment (marianfoo) [inline sources.json]: if you ready to merge, just update this

**Review by marianfoo — CHANGES_REQUESTED** [review]

File: sources.json (hunk 1 above)
Comment (flovogt) [reply sources.json]: Done

File: sources.json (hunk 2 above)
Comment (flovogt) [reply sources.json]: I've set it to next week wed. When release of 1.113 is out, i'll ping you here.

**Review by marianfoo — DISMISSED** [review]

**Review by marianfoo — APPROVED** [review]

**Comment by flovogt** [comment]
ready, @marianfoo Feel free to submit

**Comment by marianfoo** [comment]
https://bestofui5.org/#/packages/@ui5_middleware-code-coverage

**Pull request merged**